### DISCUSSION TITLE
spotube plays the wrong songs

### DISCUSSION THREAD

**Discussion by snuggle62a** [KRTirtho/spotube#963] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

It seems that some songs that are published on Spotify but not on YouTube are being played incorrectly.
For example: 
https://open.spotify.com/intl-de/track/0JXjmiB33ize81JhqupqJf
The cover image and time are displayed correctly: Here 4:15 .
However, something else is played during playback, here: 2:23:34 .

### Expected Behavior

The correct song should be played. 

### Steps to reproduce

1. I Open the app
2. I search "Reiner Bock We Are Free"
3. I start the song
4. The cover image is correct and the right time is displayed
5. A different song is played

### Operating System

Android 13

### Spotube version

v3.3.0

### Installation source

F-Droid (Android)

**Comment by snuggle62a** [comment]
The same behavior also occurs with the Flatpak version.

**Comment by snuggle62a** [comment]
Apparently Spotube does not play content from Spotify, but only from YouTube.
I have noticed that, for example, the song "When You Believe" sung by Kim Schutzius on "The Voice Of Germany" 2023 is displayed by Spotify (2:10 minutes long), but the audio part of the video is played by YouTube (8:56 minutes).

Now the question for me is, is this intentional, and if so, why can't I browse the YouTube content without logging in to Spotify?

**Reply by KRTirtho** [reply]
Yes, that's intentional. I thought we made it pretty clear in the readme 😅 

> An open source, cross-platform Spotify client compatible across multiple platforms
_utilizing Spotify's data API and YouTube (or Piped.video or JioSaavn) as an audio source,_
eliminating the need for Spotify Premium

> Now the question for me is, is this intentional, and if so, why can't I browse the YouTube content without logging in to Spotify?

This is because it's strictly a _Spotify client_. So, it only uses Spotify generated/provided playlists/album/tracks etc. Also, generally, Spotify music collection and YT Music has almost the same amount of tracks (over 100 million) so it's the same data just a bit differently bundled playlists.
Besides, for YT Music, user needs to Log in with Google account for syncing _Library_, but YouTube's data API has very limited quota. Spotify's API is very generous in those terms.

**Reply by snuggle62a** [reply]
Thank you for your explanation!
